types/mongodb/index.d.ts

```diff
@@ -1415,3 +1417,40 @@ export interface GridFSBucketWriteStreamOptions {
     wtimeout?: number,
     j?: number
 }
+
+/** http://mongodb.github.io/node-mongodb-native/3.0/api/ChangeStream.html */
+export class ChangeStream extends Readable {
+    constructor(changeDomain: Db | Collection, pipeline: Object[], options?: ChangeStreamOptions);
+
+    /** http://mongodb.github.io/node-mongodb-native/3.0/api/ChangeStream.html#close */
+    close(): Promise<any>;
+    close(callback: ChangeStreamCallback<any>): void;
```

Comment:
Can use the `MongoCallback` interface instead of the `ChangeStreamCallback` so we avoid a new interface?

Reply:
I agree. Will be fixed.